content/projects/understanding-loops/_index.md

```diff
@@ -67,20 +67,23 @@ Please note that one person can have multiple baskets. If you look at the data,
 
 ### get baskets belonging to a single customer
 
-Write a function called `get customer baskets` that has an email address as an argument and returns a list/array of all the shopping baskets that belong to the customer with that email address.
+Write a function called `get customer baskets` that takes in an email address and the data array as an argument and returns a list/array of all the shopping baskets that belong to the customer with that email address.
```

Comment:
takes in the email address and the data array as arguments. There are 2 separate arguments